tests/fiaas_deploy_daemon/deployer/kubernetes/test_deployment_deploy.py

```diff
@@ -391,7 +387,7 @@ def create_expected_deployment(config,
 
     max_surge = u"25%"
     max_unavailable = 0
-    if app_spec.replicas == 1 and app_spec.singleton:
+    if app_spec.autoscaler.min_replicas == 1 and app_spec.singleton:
```

Comment:
For the same reason as [here](https://github.com/fiaas/fiaas-deploy-daemon/pull/91/files#diff-0a3d5fdfbdba0e1dfd5133b968f65167L113), I think this should be max_replicas, or explicitly that min=max=1.

Reply:
Fixed in https://github.com/fiaas/fiaas-deploy-daemon/pull/91/commits/2ad9bab78d2e30a411424f1da6f54d37db418449